packages/expo-updates/src/UseUpdates.types.ts

```diff
@@ -0,0 +1,173 @@
+import Constants from 'expo-constants';
+
+import type { UpdatesLogEntry } from './Updates.types';
+
+export type ClassicManifest = NonNullable<typeof Constants.manifest>;
+
+export type Manifest = ClassicManifest | NonNullable<typeof Constants.manifest2>;
```

Comment:
Fine to have these here for now but when we implement https://github.com/expo/expo/pull/21880#discussion_r1156180895 in main let's remove this and just change the originals